### PR TITLE
Check pytest is installed before run tests

### PR DESCRIPTION
**Pull request by thu2004**
Check pytest is installed before run tests. 
Add --disable-extensions option when run tests.

### CODE REVIEW

File: src/test/runTest.ts [hunk 1]
```diff
@@ -1,8 +1,20 @@
 import * as path from "path";
-
+import { spawnSync } from "child_process";
+import { PythonShell } from "python-shell";
 import { runTests } from "vscode-test";
 
+async function checkPyTestIsInstalled() {
```
Comment (nickjmiller) [inline src/test/runTest.ts:6]: nitpick: this doesn't need to be async

Comment (nickjmiller) [inline src/test/runTest.ts:6]: nitpick: this doesn't need to be async

File: src/test/runTest.ts [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+    const response = spawnSync(PythonShell.defaultPythonPath, ["-m", "pytest"]);
```
Comment (nickjmiller) [inline src/test/runTest.ts:7]: This isn't quite right, the default python may not have pytest installed, but the user may have configured a different python path through the python extension.

File: src/test/runTest.ts (hunk 1 above)
Comment (thu2004) [reply src/test/runTest.ts]: Removed

File: src/test/runTest.ts (hunk 2 above)
Comment (thu2004) [reply src/test/runTest.ts:7]: The tests are run in a isolated environment so I think no users defined pytest is used.

**Pull request merged**